packages/suite/src/hooks/suite/useGuide.ts

```diff
@@ -0,0 +1,41 @@
+import { useActions, useSelector, useAnalytics } from '@suite-hooks';
+import * as guideActions from '@suite-actions/guideActions';
+import { getNodeById } from '@suite/utils/suite/guide';
+
+export const useGuide = () => {
+    const analytics = useAnalytics();
+
+    const { openGuide, openNode } = useActions({
+        openGuide: guideActions.open,
+        openNode: guideActions.openNode,
+    });
+
+    const { indexNode } = useSelector(state => ({
+        indexNode: state.guide.indexNode,
+    }));
+
+    const openNodeById = (id: string) => {
+        if (!indexNode) {
+            console.error(`Index node was not found.`);
```

Comment:
To make it clear in the logs.
```suggestion
            console.error(`Guide index node was not found.`);
```